docs/development/design/organizations.rst

```diff
@@ -0,0 +1,145 @@
+Organizations
+=============
+
+Currently we don't support organizations in the community site
+(a way to group different projects),
+we only support individual accounts.
+
+Several integrations that we support have organizations,
+where users group their repositories.
+
+Why move organizations in the community site?
+---------------------------------------------
+
+We support organizations in the commercial site,
+having no organizations in the community site makes the code maintenance difficult.
+Having organizations in the community site will make the differences between both more easy to manage.
+
+Users from the community site can have organizations in external sites from where we import their projects
+(like GitHub, Gitlab).
+Currently users have all projects from different organizations in their account.
+Having not a clear way to group/separate those.
+
+There is also a product decision.
+Supporting organizations only on code,
+but enable the feature only on the commercial site.
```

Comment:
I think we don't want this. Having organizations in the community site is a huge win from my point of view.

Reply:
Agreed. I think we'll likely need two steps, one where we merge the code, then one where we turn it on, but the end goal should definitely be having organizations on the community side.

Reply:
I would say this needs to be a two step thing. If we try to enable organization on the community side, it's only going to set this code back.

Reply:
Do you mean first move organization and then enable them? The current doc describes like that the migration, we don't enable orgs just after that.

Reply:
The doc should clarify, it doesn't answer this. I think we were disagreeing with manuel's feedback. Long term we plan on enabling organizations. Short term we are only moving the code over.

Reply:
I had an outdated view on this change. Underlying changes look good.